clients/java/client/src/main/java/org/camunda/bpm/client/variable/impl/value/DeferredFileValueImpl.java

```diff
@@ -84,4 +85,14 @@ public String toString() {
     return "DeferredFileValueImpl [mimeType=" + mimeType + ", filename=" + filename + ", type=" + type + ", isTransient=" + isTransient + ", isLoaded=" + isLoaded + "]";
   }
 
+  @Override
```

Comment:
🔧 Can we move the new getter and setter above the `toString()` method? This will be more consistent.
❓ Do you think it makes sense to include `processInstanceId` and `executionId` in the `toString()` result?

Reply:
Good points, thanks!
- Moved the new getter and setter above the `toString()` method.
- I think it can helpful when users use the `toString()` for debugging. Updated the `toString()`  to include `processInstanceId` and `executionId`